pyproject.toml

```diff
@@ -11,6 +11,8 @@ python = "^3.10"
 py-algorand-sdk = "^2.4.0"
 httpx = ">=0.23.1,<=0.28.1"
 typing-extensions = ">=4.6.0"
+algokit-transact = {url = "https://github.com/algorandfoundation/algokit-core/releases/download/python%2Falgokit_transact%401.0.0-alpha.4/algokit_transact-1.0.0a4-py3-none-manylinux_2_17_x86_64.manylinux2014_x86_64.whl"}
+algokit-algod-api = {url = "https://github.com/algorandfoundation/algokit-core/releases/download/python%2Falgod_api%401.0.0-alpha.6/algokit_algod_api-1.0.0a6-py3-none-any.whl"}
```

Comment:
Another issue here is that it will only install the wheel itself not the transitive dependencies it relies on. Unless we explicitly duplicate its transitive dependencies like pydantic on utils side - we might wanna have it published to pypi.

Reply:
Another alternative is not using the wheel at all but point to the repo and specify the tag to pull - that way it should resolve transitives

Reply:
This is solved by algorandfoundation/algokit-core#108